scripts/compare_benchmarks.py

```diff
@@ -2,15 +2,23 @@
 
 import json
 import sys
-from beautifultable import BeautifulTable
+from terminaltables import AsciiTable
 from termcolor import colored
+from scipy.stats import ttest_ind
+from array import array
 
 def format_diff(diff):
 	if diff >= 0:
 		return colored('+' + "{0:.0%}".format(diff), 'green')
 	else:
 		return colored("{0:.0%}".format(diff), 'red')
 
+def format_p_value(p_value):
+	if p_value < 0.001:
+		return colored("{0:.5}".format(p_value), 'green')
+	else:
+		return colored("{0:.5}".format(p_value), 'red')
```

Comment:
I think I would use grey if the p value is insignificant.

Reply:
done